### PR TITLE
adding final task2 and script file

### PR DESCRIPTION
**Pull request by elnemesisdivina**

### CODE REVIEW

File: main_improved.tf [hunk 1]
```diff
@@ -255,10 +273,8 @@ path.cwd is the current work directory cdw
       "sudo systemctl start httpd",
     ]
   }
-
-
 }
-
+*/
```
Comment (vladimirmukhin) [inline main_improved.tf]: Commented-out code blocks isn't a good practice in turn of the quality of code. I suggest removing provisioner block completely

File: install_el_apache.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#! /bin/bash
+sudo apt-get update
+sudo apt-get install -y apache2
+sudo systemctl start apache2
+sudo systemctl enable apache2
```
Comment (vladimirmukhin) [inline install_el_apache.sh]: Most likely it will fail. You don't need sudo. In the case of userdata, all commands are run as sudo by default.

**Review by vladimirmukhin — CHANGES_REQUESTED** [review]

**Comment by elnemesisdivina** [comment]
> _No description provided._

address comments on last review

**Review by vladimirmukhin — APPROVED** [review]
Rename main_improved.tf to main.tf and feel free to merge

**Pull request merged**